content/docs/contributing/release-process.md

```diff
@@ -476,62 +488,36 @@ page if a step is missing or if it is outdated.
        [`cert-manager/website`](https://github.com/cert-manager/website) in
        order to:
 
-       - Update the section "Supported releases" in the
-         [supported-releases](../installation/supported-releases.md) page.
        - Update the section "Supported releases" in the
          [supported-releases](../installation/supported-releases.md) page.
        - Update the section "How we determine supported Kubernetes versions" on
          the [supported-releases](../installation/supported-releases.md) page.
          In the table, set "n/a" for the line where "next periodic" is since
          these tests will be disabled until we do our first alpha.
 
-    7. **(final release only)** Open a PR to
-       [`jetstack/testing`](https://github.com/jetstack/testing) and change Prow's
-       config. To do this, take inspiration from [Maartje's PR
-       example](https://github.com/jetstack/testing/pull/397/files).
-
-    8. **(final release only)** Push a new release branch to
-       [`cert-manager/cert-manager`](https://github.com/cert-manager/cert-manager). If the
-       final release is `v1.0.0`, then push the new branch `release-1.1`:
-
-        ```bash
-        # Must be run from the cert-manager repo folder.
-        git checkout -b release-1.1 v1.0.0
-        git push origin release-1.1
-        ```
```

Comment:
note: I moved the branch creation stuff above, at the time of the first alpha. I actually think this section is totally wrong and I'm not sure we'll ever have actually run these commands because the tag for the last release (here v1.0.0) would usually be _on_ that release branch (here, release-1.0).

We wouldn't want to branch release-1.1 off release-1.0, we'd want to branch it off master.